content/manuals/security/for-admins/access-tokens.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Organization access tokens
+description: Learn how to create and manage organization access tokens
+  to securely push and pull images programmatically.
+keywords: docker hub, security, OAT, organization access token
+linkTitle: Organization access tokens (Beta)
+---
+
+{{% experimental title="Beta" %}}
+The organization access tokens feature is currently in [Beta](../../release-lifecycle.md#beta).
+{{% /experimental %}}
+
```

Comment:
```suggestion
> [!WARNING] 
> 
> Organization access tokens are not compatible with Docker Build Cloud. If you are using Docker Build Cloud, you must use personal access tokens instead. 
```

Reply:
@craig-osterhout callout for OATs compatibility with DBC

Reply:
This should probably explain that we are actively working on compatibility here. This kinda sounds like we just flat out won't support it.